src/main/java/net/minestom/server/MinecraftServer.java

```diff
@@ -85,6 +85,11 @@ public static MinecraftServer init(Configuration configuration) {
         return new MinecraftServer();
     }
 
+    @ApiStatus.Internal
+    public static ServerProcess updateProcess() {
```

Comment:
It's an internal method. Surely breaking changes don't matter at all.

Reply:
Tests probably doesn't care about some messages and values, so you can see this as a convenience method.